src/Control/Lens/Setter.hs

```diff
@@ -1043,7 +1045,7 @@ l <?= b = do
   return b
 {-# INLINE (<?=) #-}
 
--- | Modify the target of a monoidally valued by 'mappend'ing another value.
+-- | Modify the target of a semigroup value by '<>'ing another value.
```

Comment:
"`'<>'ing`" reads strangely to me. I would instead opt for this sort of phrasing:

```suggestion
-- | Modify the target of a 'Semigroup' value by using ('<>').
```